activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/QueryProcessInstancesEntityIT.java

```diff
@@ -565,4 +565,53 @@ public void shouldGetProcessInstancesFilteredByCompletedDate() {
                                          PAGED_PROCESS_INSTANCE_RESPONSE_TYPE);
     }
 
+    @Test
+    public void shouldGetProcessInstancesFilteredByInitiator() {
```

Comment:
why do we have two different assertions as part of this test?
The name of the test is quite generic `shouldGetProcessInstancesFilteredByInitiator` so i was expecting one call and one assertion.

Reply:
I will create a new method `shouldGetProcessInstancesFilteredByInitiatorList` for the second assertion

Reply:
yes that can be an option, as second option if you have concerns on creating a new IT method because will slow down the build further.. you could make clearer in the test method why there are two different goals/assertions in the same method. First option sounds cleaner